packages/jest-reporters/src/DefaultReporter.ts

```diff
@@ -120,6 +124,26 @@ export default class DefaultReporter extends BaseReporter {
     }
   }
 
+  protected __beginSynchronizedUpdate(): void {
```

Comment:
put this in `BaseReporter`?

Reply:
Is it intentional that we check `_globalConfig` every time we have a write operation?